### PR TITLE
Remove jaeger

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Remove jaeger and its refs

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactoring / Documentation
- [ ] Version change

### CODE REVIEW

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/submissionApiDelegateImpl/GetPackageInformationTest.java [hunk 1]
```diff
@@ -161,6 +160,6 @@ public void withInCorrectIDReturnForbidden() {
                 .mimeType("application/txt")
                 .isSupremeCourtScheduling(null)
                 .isAmendment(null)
-                .data(new JsonObject()).create());
+                .data(new Object()).create());
```
Comment (akroon3r) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/submissionApiDelegateImpl/GetPackageInformationTest.java:163]: are these related to the jaeger refactor? or tech debt found doing the refactor?

**Review by alexjoybc — APPROVED** [review]
